packages/@react-editor-js/core/src/editor-core.ts

```diff
@@ -8,4 +8,24 @@ export interface EditorCore {
   save(): Promise<OutputData>
 
   render(data: OutputData): Promise<void>
+
+  toggleReadOnly(): Promise<boolean>
+
+  insertBlock(type?: string, data?: BlockToolData, config?: ToolConfig, index?: number, needToFocus?: boolean): Promise<void>
+
+  updateBlock(id?: string, data?: BlockToolData): Promise<void>
+
+  deleteBlock(index?: number): Promise<void>
+
+  setToFirstBlock(position: string, offset: number): Promise<boolean>
```

Comment:
Could you provide use-case to me? Rather than exposing all API of editor-js, I think it would be better to identify use-case and apply the new abstraction 🙏🏻